code/model/DNEnvironment.php

```diff
@@ -392,8 +392,10 @@ public function canDeploy($member = null) {
 		}
 		// Must be logged in to check permissions
 
-		if(Permission::checkMember($member, 'ADMIN')) {
-			return true;
+		if ($this->Usage==='Production' || $this->Usage==='Unspecified') {
```

Comment:
Can we define a constant for the magic string 'Production', 'UAT', and 'Unspecified' for the `Usage` field?